packages/core/src/ObservableActorRef.ts

```diff
@@ -6,14 +6,13 @@ import {
   ActorContext,
   Observer
 } from './types';
-import { symbolObservable, toSCXMLEvent } from './utils';
+import { symbolObservable, toObserver, toSCXMLEvent } from './utils';
 import { Mailbox } from './Mailbox';
 import { LifecycleSignal, startSignal, stopSignal } from './actors';
 
 export class ObservableActorRef<TEvent extends EventObject, TSnapshot>
   implements ActorRef<TEvent, TSnapshot> {
   private current: TSnapshot;
-  public deferred = true;
```

Comment:
I was able to remove this completely - this was mostly needed by the `useActor` but in v5 we have lazy actors so we don't have to recompute the initial state when starting, it's all cached. Actors are also capable of queueing events before they are started - so this flag is no longer needed for anything

Reply:
Awesome!